regex/AwsCredentials.java

```diff
@@ -0,0 +1,30 @@
+package regex;
+
+public class Example {
+    public void hardCodedAccessKey() {
+        BasicAWSCredentials awsCreds = new BasicAWSCredentials("ASIAJLVYNHUWCEXAMPLE", "foo");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This looks like an AWS access key.
Don&#39;t embed access keys directly into code. The AWS SDKs and the AWS Command Line Tools allow you to put access keys in known locations so that you do not have to keep them in code. Learn more about [Best Practices for Managing AWS Access Keys](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html).